gallery/src/components/NxCodeSnippet/NxCodeSnippetPage.tsx

```diff
@@ -112,13 +123,20 @@ export default function NxCodeSnippetPage() {
         copied to the clipboard.
       </GalleryExampleTile>
 
+      <GalleryExampleTile title="Custom Sizing Example"
+                          id="nx-code-snippet-custom-sizing-example"
+                          liveExample={NxCodeSnippetSizingExample}
+                          codeExamples={nxCodeSnippetSizingCode}>
+        An <NxCode>NxCodeSnippet</NxCode> demonstrating custom sizing with the optional <NxCode>rows</NxCode> prop.
```

Comment:
The last example demonstrates the `rows` prop too, meaning that IMO the real value of this example is that it demonstrates using `rows` to create a single-line textarea.  That's what wasn't cleanly possible before this PR.  I think the name and description on this example should be oriented around that aspect.

Reply:
updated in 4e1e760